mne_bids/meg_bids.py

```diff
@@ -31,26 +35,18 @@
                  '.ds': 'CTF'}
 
 
-def _mkdir_p(path):
-    try:
-        os.makedirs(path)
-    except OSError as exc:  # Python >2.5
-        if exc.errno == errno.EEXIST and os.path.isdir(path):
-            pass
-        else:
-            raise
-
-
 def _channels_tsv(raw, fname, verbose):
     """Create channel tsv."""
-
     map_chs = defaultdict(lambda: 'OTHER')
-    map_chs.update(grad='MEGGRAD', mag='MEGMAG', stim='TRIG', eeg='EEG',
-                   eog='EOG', ecg='ECG', misc='MISC', ref_meg='REFMEG')
+    map_chs = dict(grad='MEGGRAD', mag='MEGMAG', stim='TRIG', eeg='EEG',
```

Comment:
this should use `maps_chs.update` cf `map_desc.update` below to use the `defaultdict`